_posts/2020-01-02-documenting-rcpp-packages.md

```diff
@@ -84,24 +80,14 @@ which Rcpp will translate into `#'` in the corresponding `RcppExports.R` file:
 //' @param v1 First value
 //' @param v2 Second value
 //' @return Product of v1 and v2
-//' @export
 // [[Rcpp::export]]
 double mult( double v1, double v2 ) {return v1*v2;}
 {% endhighlight %}
 
-You may notice two `export` keywords used in the comment block. The first, `@export`, tells Roxygen2
```

Comment:
Similar to the `useDynLib` point above, `@export` directives are not needed, since we're leaving `NAMESPACE` as is. I removed all such directives and the corresponding English description.

Reply:
Yep.